zerver/lib/zcommand.py

```diff
@@ -54,4 +54,6 @@ def change_mode_setting(command: str, switch_command: str,
                                             switch_command='fluid-width',
                                             setting='fluid_layout_width',
                                             setting_value=False))
+    elif command == 'status':
+        return dict(msg='Sucessfully updated status message.')
```

Comment:
Do we really need this backend change? If we're hardcoding the success message, can't we do that in the frontend itself?

Reply:
I mean, of the two server requests here --
1. The one from `zcommand.send` which sends `command=/status` (this doesn't do anything apart from receiving the success message)
2. The one from `user_status.server_update` (This is the call in which the actual work is done)

I don't think we need "1."

Reply:
Yup @abhijeetbodas2001, I was unsure about this change while creating a pr itself. To maintain the convention as followed by other <code>zcommands</code>(Creating a call to <code>/json/zcommand)</code>, I thought this might be the right way.

Reply:
Right, but those calls actually change the settings. This doesn't do anything, so I think we can just remove it. (The actual work happens in the call from `user_status.server_update`)